.github/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
         hard: 46677
 
   postgres:
-    image: postgres:11
+    image: postgres:16
```

Comment:
Do we have some documentation what is the minimum postgres version we support? I guess ideally would want to test against multiple PG versions